solutions/loading-web-fonts/globals.css

```diff
@@ -0,0 +1,5 @@
+@font-face {
```

Comment:
Wouldn't it be awesome to also use this technique on vercel.com? Looks like we currently are not.